Formula/ignition-physics5.rb

```diff
@@ -15,7 +15,7 @@ class IgnitionPhysics5 < Formula
   depends_on "cmake" => :build
 
   depends_on "bullet"
-  depends_on "dartsim@6.10.0"
+  depends_on "dartsim"
```

Comment:
This will install the latest version right (6.12.1)?

Reply:
correct https://github.com/Homebrew/homebrew-core/blob/master/Formula/dartsim.rb#L4